optimizely/logging/logger.go

```diff
@@ -70,7 +70,14 @@ func (p NamedLogProducer) Error(message string, err interface{}) {
 }
 
 func (p NamedLogProducer) log(logLevel int, message string) {
-	// prepends the name to the message
-	message = fmt.Sprintf("[%s] %s", p.name, message)
+	logLevelStrings := map[int]string{
```

Comment:
There is a cooler approach to this since we are using enums: https://programming.guide/go/define-enumeration-string.html

😄